pkg/clusteragent/orchestrator/status.go

```diff
@@ -68,22 +75,33 @@ func GetStatus(ctx context.Context, apiCl kubernetes.Interface) map[string]inter
 
 	// get cache hits
 	cacheHitsJSON := []byte(expvar.Get("orchestrator-cache").String())
-	cacheHits := make(map[string]interface{})
+	cacheHits := make(map[string]int64)
 	json.Unmarshal(cacheHitsJSON, &cacheHits) //nolint:errcheck
 	status["CacheHits"] = cacheHits
 
 	// get cache Miss
 	cacheMissJSON := []byte(expvar.Get("orchestrator-sends").String())
-	cacheMiss := make(map[string]interface{})
+	cacheMiss := make(map[string]int64)
 	json.Unmarshal(cacheMissJSON, &cacheMiss) //nolint:errcheck
 	status["CacheMiss"] = cacheMiss
+	cacheStats := make(map[string]stats)
 
 	// get cache efficiency
 	for _, node := range orchestrator.NodeTypes() {
 		if value, found := orchestrator.KubernetesResourceCache.Get(orchestrator.BuildStatsKey(node)); found {
-			status[node.String()+"sStats"] = value
+			orcStats := value.(orchestrator.CheckStats)
+			totalMiss := cacheMiss[orcStats.String()+"s"]
```

Comment:
Note: if we merge https://github.com/DataDog/datadog-agent/pull/8636 beforehand we need to update this PR and remove the `+"s"`.